servicetest/filesystem/test_filesystem.py

```diff
@@ -41,7 +46,7 @@ def logfile_path():
     Container.READONLY: False
 }
 
-@pytest.mark.usefixtures("dbus_launch", "agent", "assert_no_proxy")
+@pytest.mark.usefixtures("create_testoutput_dir", "dbus_launch", "agent", "assert_no_proxy")
 class TestFileSystem(object):
     """ This suite should do some basic testing of the Filesystem within the
         containers and that the whole chain is working properly.
```

Comment:
This test suite uses "CURRENT_DIR" for the test file (I can't comment on the line, but line 68). Should I take the opportunity to change this?